@stellar/lyra-api/webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin");
+const path = require("path");
+
+const BUILD_PATH = path.resolve(__dirname, "./build");
+
+const config = {
+  mode: "production",
+  entry: {
+    index: path.resolve(__dirname, "./src/index.ts"),
+  },
+  output: {
+    library: "lyra-api",
+    libraryTarget: "umd",
+    path: BUILD_PATH,
+    filename: "[name].min.js",
```

Comment:
this compiles the code in this subrepo with only the pertinent parts of `@lyra/api`. Comes in at a whopping 5kb 🎉

Reply:
I've been trying to test installing this package elsewhere, and i'm not 100% confident that it works atm. When I run `yarn pack`, the produced tarball still appears to want to install `@lyra/api` from npm but I'm not sure if there's a step I'm missing or something.

Reply:
@vcarl okay! I think i finally solved this. explanation in README